Android/app/src/main/java/app/intra/ui/settings/ServerApprovalDialogFragment.java

```diff
@@ -143,13 +145,24 @@ private View makeNoticeText() {
 
     final boolean showWebsite = getShowWebsite();
     final View body = showWebsite ? makeWebView(index) : makeNoticeText();
+
+    final String url = getResources().getStringArray(R.array.urls)[index];
+    final Bundle bundle = new Bundle();
+    bundle.putString(Names.SERVER.name(),
+        PersistentState.extractHostBuiltinOnly(getContext(), url));
+    final FirebaseAnalytics analytics = FirebaseAnalytics.getInstance(getContext());
+    analytics.logEvent(Names.SHOW_APPROVAL_DIALOG.name(), bundle);
     builder.setView(body)
         .setTitle(message)
         .setPositiveButton(R.string.intro_accept, (DialogInterface d, int id) -> {
-          String url = getResources().getStringArray(R.array.urls)[index];
           PersistentState.setServerUrl(getContext(), url);
+          analytics.logEvent(Names.APPROVED.name(), bundle);
```

Comment:
Maybe `TRY_ALL_ACCEPTED` and `TRY_ALL_CANCELLED`? The current names are too generic.

Reply:
Done